pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/guide.en-gb.md

```diff
@@ -76,50 +77,50 @@ Click `OK` twice:
+You can see that the local disk is now accessible and that the Windows disk is drive `(E:)` (spreading on two software RAID disks with Mirrored volume type).
 
 ![disk_import_sync](images/disk_import_sync.png){.thumbnail}
 
 > [!primary]
-> In this example, the volume status is "Resynching" because the server was hard-rebooted in rescue mode. This is the normal behaviour and it is not caused by the rescue itself.
-> This will not affect data on the volume and resynchronisation will continue after the server is rebooted on its normal OS.
+> In this example the volume status is "Resynching" because the server was hard-rebooted in rescue mode. This is an expected status and it is not caused by the rescue itself.
+> This will not affect data on the volume and resynchronisation will continue once the server is rebooted on its installed OS.
 
 > [!warning]
 >
 > You must then use the path of your local Windows directory (here it is `E:\Windows`) when browsing to find the "SAM" configuration file below.
 
 You can now reset the password by following the instructions below.
 
-#### B) Resetting the pasword <a name="sectionB"></a>
+#### B) Resetting the password <a name="sectionB"></a>
 
-To reset passwords, the NTPWEdit tool is required.<br>
-Once you are connected via Remote Desktop, open the browser and download it from its official website.<br>
-Navigate to the folder where the downloaded ZIP file is located and extract its content.<br>
+To reset passwords, the NTPWEdit tool is required.
+Once you are connected via Remote Desktop, open the web browser (MS Edge) and download the tool from the [official website](http://www.cdslow.org.ru/files/ntpwedit/ntpwed07.zip).
+Navigate to the folder where the downloaded ZIP file is located and extract its content.
 Next, open the `ntpwedit64` executable to start the application.
 
 In this interface, you can manipulate the SAM file in order to clear the admin user's password.
 
-You must browse the local Windows drive to find your server's SAM file.
+You must browse the local Windows drive to find your system's SAM file.
 
-Click the three dots button (`...`{.action}) to browse the drive containing your server's local Windows drive
+Click the three dots button (`...`{.action}) to browse the drive containing your server's local Windows folder.
 
-It usually is `Windows (E:\)`
+It usually is the `Windows (E:\)` drive.
 
 ![ntpwedit1](images/ntpwedit_1.png){.thumbnail}
 
 Browse to `E:\WINDOWS\SYSTEM32\CONFIG\`
 
-Open the SAM file to display the list of users by clicking `Open`{.action}.
+Select and open the SAM file to display the user accounts by clicking `Open`{.action}.
 
 ![ntpwedit_sam](images/SAM.png){.thumbnail}
 
-The relevant user will either be "admin" or "Administrator", depending on the Windows version. If both are present, choose "admin". Then click `Change password`{.action}.
+Select the user account "admin" and click on `Change password`{.action}.
```

Comment:
I believe using "click" and not "click on" was deliberate:
> Style guides for technical writers generally recommend using click transitively (for example: click the button), but intransitive use with on (click on the icon) is also widespread. The style guides do accept the use of in in phrases like click in the field. 
> https://en.wiktionary.org/wiki/click

Reply:
Both OK